experimental/freetextfetcher/store.go

```diff
@@ -48,9 +48,9 @@ func (fts *freetextStore) StopFetching(userID string) error {
 	return fts.client.KV.Delete(fts.getKey(userID))
 }
 
-func (fts *freetextStore) ShouldProcessFreetext(userID, fetcherID string) (bool, string, error) {
+func (fts *freetextStore) ShouldProcessFreetext(userID, fetcherID string) (shouldProcess bool, payload string, err error) {
```

Comment:
Curious what linting rule triggered this change? Named return values are usually exceptional.

Reply:
https://go-critic.github.io/overview#unnamedresult is the rule in question. Maybe enabling `opinionated` wasn't a good idea: https://github.com/mattermost/mattermost-plugin-api/blob/d4df2d1e8f88e7d892eac4f7aac83c87dd22dde8/.golangci.yml#L47